### PR TITLE
Removed Docker build references from Online docs

### PR DESCRIPTION
**Pull request by ahardin-rh**
@tiwillia PTAL

Addresses https://bugzilla.redhat.com/show_bug.cgi?id=1476914#c5

### CODE REVIEW

File: creating_images/guidelines.adoc [hunk 1]
```diff
@@ -124,7 +119,7 @@ them, where possible, so they do not end up written to a layer.
 
 In addition, performing multiple commands in a single `RUN` statement reduces
 the number of layers in your image, which improves download and extraction time.
-
+endif::[]
```
Comment (tiwillia) [inline creating_images/guidelines.adoc]: I don't see any reason for this to be hidden for online. These are part of the recommendations on how to build images to use in OpenShift. This section does not describe using the docker build strategy in Openshift.

@sferich888 would you like to confirm?

File: creating_images/guidelines.adoc [hunk 2]
```diff
@@ -138,34 +133,29 @@ For example, if you are working on a *_Dockerfile_* that contains an `ADD`
...
 ADD myfile /test/myfile
 RUN yum -y install mypackage && yum clean all -y
 ----
-====
 
 Then each time you changed *_myfile_* and reran `docker build`, the `ADD`
 operation would invalidate the `RUN` layer cache, so the `yum` operation would
 need to be rerun as well.
-
+endif::[]
```
Comment (tiwillia) [inline creating_images/guidelines.adoc]: Same with this. This describes running the `docker build` command outside of OpenShift.

**Review by tiwillia — CHANGES_REQUESTED** [review]
Looks good besides the guidelines document. I don't think that needs to be modified at all, since its describing how to properly build images for use with OpenShift. Its not describing the docker build strategy in OpenShift at all.

@sferich888 can you confirm?

**Comment by tiwillia** [comment]
Discussed with @sferich888 externally, the guidelines doc doesn't need to be touched in this PR.

**Comment by ahardin-rh** [comment]
@tiwillia Thanks! Updated.

**Comment by ahardin-rh** [comment]
No revision history needed

**Pull request merged**